### ISSUE TITLE
TS Lint depecreated?

### ISSUE THREAD

**Issue by naknode** [a7urag/node-express-mysql-typescript-api-boilerplate#3]
I am looking for exactly this solution. Node.js API with TypeScript and TypeORM support. Simple. I see you are using TS Lint -- is that not going to be discontinued soon?

**Comment by a7urag** [comment]
Yes, it is going to be discontinued soon. But, in this project `tslint-config-airbnb` is used which can be easily replaced with eslint and `tslint-config-airbnb`.
